docs/source/indepth_tutorial/mpi-evaluator.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<p>this is a very nice and clear write up!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/308/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Thanks!